src/aat/resources/features/F-056/F-056.feature

```diff
@@ -52,4 +52,18 @@ Feature: F-056: Submit event creation as a Citizen
     And the response [includes the case detail for the updated case, along with a HTTP 200 OK]
     And the response has all other details as expected
 
+  @S-582
+  Scenario: must update successfully the respective fields with ACL permissions for a Citizen
+    Given a user with [an active Citizen profile in CCD with a specific variation of ACLs on a case type]
```

Comment:
this scenario has two users (a citizen and a privileged user), so both users should be mentioned here. The specification for each user should also be more specific, eg.

```
Given a user with [an active citizen profile in CCD without update permissions for document fields in a given case type]
And a user with [an active profile in CCD with full permissions in the same given case type]
```